docs/content/deployment/run-monitoring.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Run Monitoring | Dagster
+description: Features for manging run failures in Dagster.
+---
+
+# Run Monitoring
+
+Dagster can detect hanging runs and restart crashed run workers. Run monitoring is currently only supported with:
```

Comment:
Run monitoring is currently only supported on instances using one of the following run launchers: